.github/workflows/bft-integration-tests.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         run: go install -mod=mod github.com/onsi/ginkgo/ginkgo@v1.11.0 && go get github.com/onsi/gomega/...
 
       - name: Vendor
-        run: go mod tidy && go mod vendor
+        run: go mod tidy -go=1.16 && go mod tidy -go=1.17 && go mod vendor
```

Comment:
I don't understand why we need both of them?

Reply:
For maintaining compatibility with go 1.16 as well. If we use only 1.17 it fails